lib/src/code_theme/code_theme.dart

```diff
@@ -5,9 +5,9 @@ import 'code_theme_data.dart';
 class CodeTheme extends InheritedWidget {
   const CodeTheme({
     required this.data,
-    Key? key,
-    required Widget child,
-  }) : super(key: key, child: child);
+    super.key,
+    required super.child,
+  });
```

Comment:
1. super.key
2. Everything required on this class.
3. Everything required on super.
4. Everything optional on this class.
5. Everything optional on super.